applications_tests/lethe-fluid/high_pe_stab_gls.output

```diff
@@ -28,12 +28,12 @@ Steady iteration:        3/3
    Volume of triangulation:      1
    Number of thermal degrees of freedom: 4225
 L2 error velocity : 5.55112e-17
-L2 error temperature : 0.000212391
+L2 error temperature : 0.00112348
```

Comment:
We will need to double-check the high Pe tests.

Reply:
Yeah, make sure that they converge. Here you seem to have affected the order of convergence significantly. It was already too high though, so might not be the worst thing, but still, double check that it is still consistant.